package.json

```diff
@@ -18,6 +18,7 @@
     "site:build": "dumi build",
     "site:deploy": "gh-pages -d site",
     "start": "npm run dev",
+    "sync:other-npm": "node -e 'require(\"./scripts/sync-tnpm-with-cnpm\")()'",
```

Comment:
- `sync-cnpm-with-tnpm` => `sync-cnpm-and-tnpm`
- Add `postpublish` script to execute `sync:other-npm`